cirq/linalg/transformations.py

```diff
@@ -388,10 +388,15 @@ def subwavefunction(wavefunction: np.ndarray,
     of kets like  $|\psi\rangle$ = $|x\rangle \otimes |y\rangle$, and
     $|x\rangle$ is defined over the subset `keep_indices` of k qubits, then
     this method will factor $|\psi\rangle$ into $|x\rangle$ and $|y\rangle$ and
-    return $|x\rangle$.  Note that $|x\rangle$ is not unique, because $(e^{i
-    \theta} |y\rangle) \otimes (|x\rangle) = (|y\rangle) \otimes (e^{i \theta}
-    |x\rangle)$ . This method randomizes the global phase of $|x\rangle$ in
-    order to avoid accidental reliance on it.
+    return $|x\rangle$. Note that $|x\rangle$ is not unique, because
+
+    $$
+        (e^{i \theta} |y\rangle) \otimes (|x\rangle)
+        = (|y\rangle) \otimes (e^{i \theta} |x\rangle)
+    $$
+
```

Comment:
This is certainly an improvement, but parsing LaTeX is still a pain when looking at code (e.g. in github). In this case, the pain may be completely avoided by spelling this out in English:

> ..., because scalar multiplication may be absorbed by any factor of a tensor product.

Reply:
done.

Reply:
Agreed